view/business-process-data-forms.js

```diff
@@ -31,7 +32,13 @@ exports.step = function () {
 	insert(_if(and(eq(guideProgress, 1),
 		eq(dataForms._progress, 1)),
 		div({ class: 'user-next-step-button' },
-			a({ href: '/documents/' }, _("Continue to next step"))),
+			_if(eq(applicableDocs, 0),
+				_if(eq(businessProcess.costs._paymentWeight, 0),
+					a({ href: '/submission/' }, _("Continue to next step")),
```

Comment:
You can just wrap the href value in `_if`s. Like:

```javascript
a({ href: _if(…) }, _("Continue to next step")))
```

This will limit the copy paste.